### PR TITLE
Act 63 create api and service for event moderation

### PR DESCRIPTION
**Pull request by Sang-Tan**

### CODE REVIEW

File: src/main/java/com/dutact/web/features/event/admin/controllers/AdminEventController.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    public ResponseEntity<?> getEvent(@PathVariable Integer id) {
+        Optional<EventDto> event = eventService.getEvent(id);
+        if (event.isEmpty()) {
+            return ResponseEntity.notFound().build();
+        }
+
+        return ResponseEntity.ok(event);
+    }
+
+    @PutMapping("/{id}/status")
```
Comment (TranSiTien) [inline src/main/java/com/dutact/web/features/event/admin/controllers/AdminEventController.java:37]: If status change to REJECTED didn't it need reason too?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -32,4 +32,6 @@ spring.mail.properties.mail.smtp.auth=true
 spring.mail.properties.mail.smtp.starttls.enable=true
 
 # Flyway Migration
-spring.flyway.locations=classpath:db/migration
\ No newline at end of file
+spring.flyway.locations=classpath:db/migration
+
+sprv.rbac.admin=ADMIN
```
Comment (TranSiTien) [inline src/main/resources/application.properties]: what is sprv?

Comment (TranSiTien) [reply src/main/resources/application.properties]: and where do this using, I can't find reference

File: src/main/java/com/dutact/web/features/event/admin/controllers/AdminEventController.java (hunk 1 above)
Comment (Sang-Tan) [reply src/main/java/com/dutact/web/features/event/admin/controllers/AdminEventController.java:37]: the `EventStatus` object contain reject reason for `Rejected`

File: src/main/resources/application.properties (hunk 2 above)
Comment (Sang-Tan) [reply src/main/resources/application.properties]: forgot to delete when testing

**Review by TranSiTien — APPROVED** [review]
LGTM

**Pull request merged**